src/app/config/config.ts

```diff
@@ -2,7 +2,8 @@ export const config = {
     'interviewRounds': [{'name': 'First Round', 'value': 'first_round'}, {'name': 'Second Round', 'value': 'second_round'}, {'name': 'Third Round', 'value': 'third_round'}],
     'userType': ['Admin', 'Hr', 'Guest'],
     'mobileNoPrefix': '+91',
-    'roles':  {'HR': ['/core/setting/emailtemplate', '/core/setting/emailvariable', '/core/setting/tagsetting', '/core/setting/automaticTags', '/core/setting/jobProfileTags', '/core/inbox', '/core/setting/resetPassword'], 'Guest': [ '/core/inbox', '/core/setting/resetPassword']},
+    'titles': {'/core/dashboa': 'Dashboard', '/core/setting' : 'Settings', '/core/inbox': 'Inbox', '/core/inbox/e': 'Inbox'},
```

Comment:
looks like a spelling mistake '/core/dashboa'?

Reply:
no sir. i am using substr() for all urls like /core/settings/imap , /core/settings/smtp, /core/settings/userList, /core/settings/roles. they all are belongs to /core/settings. So it cut it and make condition in one line this.title = config['titles'][event['url'].substr(0, 13)];